core/mondoo-linux-security.mql.yaml

```diff
@@ -1123,12 +1121,9 @@ queries:
         ```
   - uid: mondoo-linux-security-audit-log-storage-size-is-configured
     title: Ensure audit log storage size is configured
-    impact: 80
+    impact: 40
     mql: |
-      file("/etc/audit/auditd.conf").exists
```

Comment:
We could also make this a filter of the query itself so the check only runs if the config is there. On the other hand this policy already requires `auditd` to be installed, so the config must exist anyway.

Reply:
i have introduced two groups to solve it